com.budgetwithbuckets.Buckets.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: com.budgetwithbuckets.Buckets
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: buckets.sh
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  - name: bucketstar
+    buildsystem: simple
+    sources:
+      - type: extra-data
+        filename: buckets.tar.gz
+        only-arches:
+          - x86_64
+        url: https://github.com/buckets/application/releases/download/v0.71.1/Buckets-0.71.1.tar.gz
+        sha256: 4c5cb0812965ed75dd5c22abf7fcc6fa10d9e8ad34c5b633e575a8790dc1c651
+        size: 75530266
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - mkdir buckets
+          - tar xf buckets.tar.gz --strip-components=1 -C ./buckets
+          - rm -f buckets.tar.gz
+    build-commands:
+      - install apply_extra /app/bin/apply_extra
+
+  - name: buckets
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 com.budgetwithbuckets.Buckets.desktop -t /app/share/applications
+      - |
+        set -x
+        for dir in icons/*; do
```

Comment:
You can just install the 512x512 one, appstream-compose will create 64px and 128px icons from it